README.md

```diff
@@ -77,17 +77,13 @@ Using effects brings several advantages:
 
 Additionally, modern operating systems provide high-performance alternatives to the old Unix `select` call.
 For example, Linux's io_uring system has applications write the operations they want to perform to a ring buffer,
-which Linux handles asynchronously.
-
-Due to this, many OCaml users will want to rewrite their IO code.
-It would be very beneficial to use this opportunity to standardise a single concurrency API for OCaml,
-and we hope that Eio will be that API.
```

Comment:
Hi! Line 83-84 of the README.md was one of my favorite parts in Eio!

> It would be very beneficial to use this opportunity to standardise a single concurrency API for OCaml,
> and we hope that Eio will be that API.

I'm curious why it was removed. I'd be interested in learning more!

Not sure how notification work on GitHub on closed PR, so I'll mention you directly @talex5 with the hope that someone from the eio team see this comment. Thank you!

Reply:
That text was written before the various other schedulers appeared. To someone who doesn't know the history, it might sound like we're trying to fight with them.

Reply:
Got it. I also read the Eio 1.0 [blog post](https://tarides.com/blog/2024-03-20-eio-1-0-release-introducing-a-new-effects-based-i-o-library-for-ocaml/) today. That's very helpful context, thank you very much!